class/defaults.yml

```diff
@@ -57,6 +57,10 @@ parameters:
         enabled: false
         secretNamespace: ${crossplane:namespace}
         postgres:
+          # bucket_region: 'lpg' || 'ch-gva-2'
+          bucket_region: ''
```

Comment:
Do we really need the region here? If I'm not wrong there's only one endpoint for buckets for Exoscale and Cloudscale. The region is necessary. You might want to check that though.

Reply:
yes we need, I spoke about it with Simon and Fabian and they all want it to be cloud specific, so I need to make it parameter that will be specified in project syn for deployment